evm-contracts/src/v0.6/dev/SimpleAccessControl.sol

```diff
@@ -36,7 +36,7 @@ contract SimpleAccessControl is AccessControllerInterface, Owned {
     override
     returns (bool)
   {
-    return accessList[_user];
+    return checkEnabled ? accessList[_user] : true;
```

Comment:
Sorry to nitpick, but I think `accessList[_user]  || checkEnabled` would be cleaner

Reply:
Also, there's now a redundant || !checkEnabled in the checkAccess modifier.

And does checkAccess have to be virtual or could we get rid of that?